Example/iOS Example/AppDelegate.swift

```diff
@@ -43,6 +43,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 return
         }
         PiwikTracker.configureSharedInstance(withSiteID: siteid, baseURL: baseurl)
+        PiwikTracker.shared?.logger?.minLevel = .info
```

Comment:
My experience from looking at a lot of iOS/Mac code is a pattern of overusing singletons. I am just trying to avoid adding to this problem :)

I personally do not see any reason for the framework to provide a singleton. A user that really want to use the singleton pattern in their code can just declare a public variable in the `AppDelegate` or introduce a `Tracker` class with a shared instance or a global variable.

Up to you. I am just on a personal crusade to reduce the use of singletons.